NAMESPACE

```diff
@@ -1,315 +1,212 @@
-import(stats)
-importFrom(lattice,
-           xyplot,
-           qqmath)
-importFrom(car,
-           powerTransform,
-           bcPower)
-importFrom(Hmisc,
-           wtd.quantile)
-import(zoo)
-importFrom(graphics,plot)
-import(latticeExtra)
-import(polynom)
-import(reshape)
+# Generated by roxygen2: do not edit by hand
 
-## Exports from imported packages
-export(simulate,fitted,residuals,predict,update,coef)
-export(xyplot)
-# export(summary,print)
-export(plot)
-
-## CORE FUNCTIONS
-
-export(hydromad)
-S3method(print, hydromad)
-S3method(fitted, hydromad)
-S3method(residuals, hydromad)
-S3method(observed, hydromad)
-S3method(vcov, hydromad)
-S3method(logLik, hydromad)
-S3method(deviance, hydromad)
-S3method(str, hydromad.runlist)
-export(isValidModel)
-S3method(isValidModel, default)
-S3method(isValidModel, hydromad)
-S3method(isValidModel, runlist)
-S3method(isValidModel, tf)
-
-S3method(predict, hydromad)
-
-S3method(update, hydromad)
-
-S3method(coef, hydromad)
-
-export(absorbScale)
-S3method(absorbScale, hydromad)
-
-S3method(simulate, hydromad)
-
-export(parameterSets)
-export(rotatedSampling)
-
-S3method(summary, hydromad)
-S3method(summary, hydromad.runlist)
-S3method(print, summary.hydromad)
-S3method(print, summaryWithBreaks.hydromad)
-
-S3method(plot, hydromad)
-S3method(xyplot, hydromad)
-S3method(xyplot, hydromad.runlist)
-S3method(qqmath, hydromad)
-S3method(tsdiag, hydromad)
-
-
-## ROUTING MODELS
-
-export(tryModelOrders)
-S3method(summary, tryModelOrders)
-
-export(makePrefilter, ## remove this?
-       estimateDelay,
-       estimateDelayFrac,lagFrac
-       )
-
-export(tfParsConvert)
-
-export(armax.sim,
-       ssg.armax,
-       normalise.armax,
-       armax.ls.fit,
-       armax.sriv.fit,
-       armax.inverse.fit,
-       armax.inverse.sim)
-useDynLib(hydromad, sriv_system)
-useDynLib(hydromad, inverse_filter)
```

Comment:
This PR appears to remove `inverse_filter` from the namespace. But it's still used in `R/armax.inverse.sim.R`. Does that work? Is it the right thing to do?

Has rd2roxygen basically remove anything from NAMESPACE that does not have API documentation?

Reply:
Yes, I had to go back and double check the original NAMESPACE to see which files needed to be exported.

The useDynLib() exports are gone, which is especially frustrating because the lack of those those C to R exports are currently breaking the tests.

Reply:
This is presumably fine now?

Reply:
Yup, all the useDynLib exports are in NAMESPACE.